### DISCUSSION TITLE
As in, in my `.bashrc` I put:

### DISCUSSION THREAD

**Discussion by inmortalbobz** [microsoft/debugpy#1267] [Enhancements]
As in, in my `.bashrc` I put:

```bash
export DEBUGPY_EXCEPTION_FILTER_USER_UNHANDLED=1
```
and then restart VSCode?

_Originally posted by @rijobro in https://github.com/microsoft/debugpy/issues/550#issuecomment-784007595_

**Comment by int19h** [comment]
If you do it that way, I think you'll need to log out and relogin for it to take effect.
